Source/OxyPlot/Series/BoxPlotSeries.cs

```diff
@@ -94,6 +97,18 @@ public BoxPlotSeries()
         public double MedianThickness { get; set; }
 
         /// <summary>
+        /// Gets or sets the size of the mean point.
+        /// </summary>
+        /// <remarks>This property is only used when ShowMeanAsDot = true.</remarks>
+        public double MeanPointSize { get; set; }
```

Comment:
Yeah, that's what I was also thinking about, but I guess for now the possibility to set some different styles and sizes should be enough.

It was just a suggestion, as I'm myself not 100% sure it is clear enough, but this would be more or less one enum to represent the following things:

```c#
public enum WhatEverStyle {
    None, // Pretty clear I guess
    Line, // draw a simply line
    Point, // draw a dotted line
    Dash, // draw a dashed line
    Circle, // draw a circle marker
    Square, // draw a square marker
    Star // draw a star marker
}
```

this would be the clearest way to prevent mean and median from being drawn as a line and a marker at the same time. and you don't have to set two properties every time you want to display it as a circle

from what I did until now with BoxPlot charts I can tell you that outliers should definitively be separated style/size wise from the rest. median and mean could probably be the same size but should always have different styles (for clear reasons).